src/style.css

```
@@ -0,0 +1,41 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  align-content: center;
+}
+
+.stars__star {
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars--0 > .stars__star:nth-child(n+1) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--1 > .stars__star:nth-child(n+2) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--2 > .stars__star:nth-child(n+3) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--3 > .stars__star:nth-child(n+4) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--4 > .stars__star:nth-child(n+5) {
+  background-image: url(./images/star.svg);
+}
```

You can group these. and no need to combine
```suggestion
.stars--1 :nth-child(1),
.stars--2 :nth-child(-n+2),
.stars--3 :nth-child(-n+3),
.stars--4 :nth-child(-n+4),
.stars--5 :nth-child(-n+5), {
  background-image: url(./images/star.svg);
}
```